pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -133,6 +145,10 @@ type Cloud struct {
 	resourceRequestBackoff   wait.Backoff
 	metadata                 *InstanceMetadata
 
+	// Clients for vmss.
+	VirtualMachineScaleSetsClient   compute.VirtualMachineScaleSetsClient
```

Comment:
what is VirtualMachineScaleSetsClient used for?

Reply:
It is not used yet in this PR, but will be used in next one.